airbyte-webapp/src/components/StreamTestingPanel/StreamTestingPanel.tsx

```diff
@@ -19,6 +20,12 @@ export const StreamTestingPanel: React.FC<unknown> = () => {
     stream: selectedStream.name,
     config: configJson,
   });
+  const [logsFlex, setLogsFlex] = useState(0);
+
+  const handleLogsTitleClick = () => {
+    // expand to 50% if it is currently less than 50%, otherwise minimize it
+    setLogsFlex((prevFlex) => (prevFlex < 0.5 ? 0.5 : 0));
```

Comment:
Minor nitpick: my personal feeling would be that we should always collapse it if it's somewhat open and open to 50% if it's completely closed. It feels a bit more aligned with what I personally would expect, but not having extremely strong feelings around that.